lighthouse-plugin-publisher-ads/package.json

```diff
@@ -31,7 +31,7 @@
     "yargs": "^16.1.1"
   },
   "peerDependencies": {
-    "lighthouse": "8.4.0"
+    "lighthouse": "8.6.0"
```

Comment:
I think you'll need to run yarn and update yarn.lock

Reply:
Yep! Done now.